### PR TITLE
UI changes for rally widget

### PR DESCRIPTION
**Pull request by dhivahar19**
Rally widget has been implemented and in Hygieia dashboard

### CODE REVIEW

**Comment by dhivahar19** [comment]
Please find the attachment for this pull request
[rally-hygieia screenshot.docx](https://github.com/capitalone/Hygieia/files/1837719/rally-hygieia.screenshot.docx)

**Comment by dhivahar19** [comment]
@nireeshT - We have different data points in Rally compared to other Project Management tools. If we have to follow the feature widget structure,we have to change our entire source codes to support it. Do you want us to proceed or we can have this Rally as a separate widget?

**Comment by dhivahar19** [comment]
@nireeshT - I have made changes in that file and made rally widget as standalone.

**Comment by nireeshT** [comment]
@dhivahar19  awesome

File: UI/src/components/widgets/rally/view.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                templateUrl: 'components/widgets/rally/rallyBuildDetail.html',
+                size: 'lg',
+                resolve: {
+                    resultData: function() {
+                        return resultData;
+                    }
+                }
+            });
+        }
+        ctrl.rallyUrl = "https://idp.aa.com/idp/startSSO.ping?PartnerSpId=sso.rallydev.com";
```
[the remark this answers is not in the document]
Comment (dhivahar19) [reply UI/src/components/widgets/rally/view.js]: @nireeshT - It has been removed.

**Comment by nireeshT** [comment]
@dhivahar19  update branch

**Comment by nireeshT** [comment]
@dhivahar19  Can you provide documentation (Readme document)  for rally collector with properties file changes

**Pull request merged**